src/main/java/mate/jdbc/model/Manufacturer.java

```diff
@@ -0,0 +1,45 @@
+package mate.jdbc.model;
+
+public class Manufacturer {
+    private Long id;
+    private String name;
+    private String country;
+
+    public Manufacturer() {
+    }
+
+    public Manufacturer(String name, String country) {
```

Comment:
Currently, there are no checks on the values passed to the setters or constructors. To ensure the integrity of your data, consider adding validation logic to prevent invalid states. For example, name and country should not be allowed to be set to null or an empty string.